bot/cogs/hacktoberstats.py

```diff
@@ -0,0 +1,184 @@
+import re
+import typing
+from collections import Counter
+from datetime import datetime
+
+import aiohttp
+import discord
+from discord.ext import commands
+
+
+class Stats:
+    def __init__(self, bot):
+        self.bot = bot
+        self.channelID = (496432022961520650)  # Hardcode #event-hacktoberfest channel ID
+
+    @commands.command(
+        name="stats",
+        aliases=["getstats", "userstats"],
+        brief="Get a user's Hacktoberfest contribution stats",
+    )
+    async def getstats(self, ctx, *args):
+        """
+        Query GitHub's API for PRs created by a GitHub user during the month of October that 
+        do not have an 'invalid' tag
+
+        For example:
+            !getstats heavysaturn
+
+        If a valid username is provided, an embed is generated and posted to the channel
+
+        Otherwise, post a helpful error message
+
+        The first input argument is treated as the username, any additional inputs are discarded
+        """
+        username = args[0]
+        PRs = await self.getoctoberprs(username)
+
+        postchannel = self.bot.get_channel(self.channelID)
+        if PRs:
+            statsembed = self.buildembed(username, PRs)
+            await postchannel.send('Here are some stats!', embed=statsembed)
+        else:
+            await postchannel.send(f"No October GitHub contributions found for '{username}'")
+
+    def buildembed(self, username: str, PRs: typing.List[dict]) -> discord.Embed:
```

Comment:
This function should be called `build_embed` and the `PRs` argument should be lowercase.